android/tangram/src/com/mapzen/tangram/MapController.java

```diff
@@ -747,6 +768,18 @@ public void queueSceneUpdate(String componentPath, String value) {
     }
 
     /**
+     * Enqueue a scene component update with its corresponding YAML node value
+     * @param componentPath The YAML component path delimited by a '.' (example "scene.animated")
+     * @param value A YAML valid string (example "{ property: true }" or "true")
+     */
+    public void queueSceneUpdate(ArrayList<SceneUpdate> sceneUpdates) {
+        checkPointer(mapPointer);
+        for(SceneUpdate sceneUpdate : sceneUpdates) {
+            nativeQueueSceneUpdate(mapPointer, sceneUpdate.path, sceneUpdate.value);
```

Comment:
Don't we want to make it a single call here `nativeQueueSceneUpdates(long mapPtr, String[] sceneUpdatePaths, String[] sceneUpdateValues)`? It would reduce jumping between platform code and core library.